dockerfiles/examples/local/hpcc-localfile/templates/NOTES.txt

```diff
@@ -0,0 +1,21 @@
+{{ $dllPvcName := printf "dll-%s-pvc" (include "hpcc-localfile.fullname" $) }}
+{{ $daliPvcName := printf "dali-%s-pvc" (include "hpcc-localfile.fullname" $) }}
+{{ $dataPvcName := printf "data-%s-pvc" (include "hpcc-localfile.fullname" $) }}
+
+Persistent Volume Claims created: {{ $dllPvcName }}, {{ $daliPvcName }}, {{ $dataPvcName }}
```

Comment:
I wonder whether these lines should be prefixed with # - so you can capture the output into a yaml that you can use directly